web/src/components/PassiveTreeViewer/index.tsx

```diff
@@ -138,6 +143,20 @@ export function PassiveTreeViewer({ urlTrees }: PassiveTreeViewerProps) {
           <label className={classNames(styles.label)}>
             {urlTrees.length > 0 && (urlTrees[curIndex].name || "Default")}
           </label>
+          {activeGemLinks.length > 0 && (
```

Comment:
Would prefer if this was it's own component and instead expose the currently selected passive tree using recoil state